martin/src/pg/pool.rs

```diff
@@ -42,21 +45,26 @@ impl PgPool {
 
         let conn = get_conn(&pool, &id).await?;
         let pg_ver = get_postgres_version(&conn).await?;
-        if pg_ver < MINIMUM_POSTGRES_VER {
-            return Err(PostgresqlTooOld(pg_ver, MINIMUM_POSTGRES_VER));
+        if pg_ver < MINIMUM_POSTGRES_VERSION {
+            return Err(PostgresqlTooOld(pg_ver, MINIMUM_POSTGRES_VERSION));
         }
-        if pg_ver < RECOMMENDED_POSTGRES_VER {
-            warn!("PostgreSQL {pg_ver} is older than the recommended {RECOMMENDED_POSTGRES_VER}.");
+        if pg_ver < RECOMMENDED_POSTGRES_VERSION {
+            warn!("PostgreSQL {pg_ver} is older than the recommended {RECOMMENDED_POSTGRES_VERSION}.");
         }
 
         let postgis_ver = get_postgis_version(&conn).await?;
```

Comment:
lets move error cases before the warning cases. This way we won't print a warning and then error out